frontend/src/lib/components/UniversalSearch/UniversalSearch.scss

```diff
@@ -1,7 +1,7 @@
 @import '../../../styles/breakpoints';
 
 .universal-search-box {
-    width: 15rem;
+    max-width: 15rem;
     height: 100%;
```

Comment:
wonder if these should be in global tailwind style? `h-full cursor-pointer`

Reply:
They certainly should be but I'm tempted to leave it until we hit the design system work as then we can take advantage of thinks like `md:` for breakpoints and solve it in one fell swoop ;)